scripts/run-resmoke.sh

```diff
@@ -42,7 +42,8 @@ cat << EOF >> "${SENTINEL_XML}"
 EOF
 
 # Add the mongo dir to the end of the path so we can find mongo executables
-export PATH="${PATH}:${MONGO_DIR}"
+export PATH+=":${MONGO_DIR}"
```

Comment:
remove this line?

Reply:
It allows people to run with local builds. The `bin` form works with our binary archives.

Reply:
Making it easy for engineers to be able to run with local builds sounds useful to me. I think this is a quirk of how we're running resmoke.py from a location that isn't the top-level of the mongodb/mongo repository - it is going to check for `mongod`, etc. in the top-level of the mongodb/genny repository.